cpp/src/randomforest/randomforest.cu

```diff
@@ -341,6 +347,102 @@ std::vector<unsigned char> save_model(ModelHandle model) {
   return bytes_info;
 }
 
+void check_concat_tl_mod(ModelHandle concat_tree_handle,
+                         std::vector<ModelHandle*> treelite_handles) {
+  size_t concat_forest;
+  size_t single_handle;
+  int total_num_trees;
+  for (int tl_num = 0; tl_num < treelite_handles.size(); tl_num++) {
```

Comment:
how about `for (int handle_idx` or `int forest_idx` as a different name? tl_num sounds like the number of tls